pkg/model/types.go

```diff
@@ -16,7 +16,8 @@ type Metadata struct {
 	Description     string            `json:"description,omitempty"`
 	Tags            []string          `json:"tags,omitempty"`
 	Labels          map[string]string `json:"labels,omitempty"`
-	Format          string            `json:"format"`
+	Format          string            `json:"format,omitempty"`
+	Framework       string            `json:"framework,omitempty"`
 	Metrics         []Metric          `json:"metrics,omitempty"`
 	Hyperparameters []Hyperparameter  `json:"hyperparameters,omitempty"`
```

Comment:
In [spec definition](https://github.com/caicloud/ormb/commit/734593e4f5c3d26b0f9d6de55986735a49242aa9\#diff-5441b3fdbf500dd8b6f82d339f005276R88),  I see some field is `REQUIRED`, eg: `format` field , these field SHOULD limit by `json:"format,omitempty" required: "true"`.

Reply:
required: "true" is it really work? Who will check the tag to verify it is a required field?

Reply:
Forget it...... it guess json.Unmarshal will check it, but it is not.

Find other method to verify it in the future.

Reply:
Yeah, I think so. We should have something like https://github.com/jessevdk/go-flags to support the feature.